<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MakeGrid</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Rows</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="rows_spin">
         <property name="styleSheet">
          <string notr="true">QSpinBox {
	max-height: 11px;
	min-height: 11px;
    min-width: 60px;
    max-width: 60px;
    padding-right: 10px;
    background-color: lightgray;
    padding-top: -2px;
    padding-bottom: -2px;
}

QSpinBox::up-button {
  subcontrol-origin: padding;
  subcontrol-position: right center;
  left: 7px;
  height: 18px;
  width: 18px;
}

QSpinBox::down-button {
  subcontrol-origin: content;
  subcontrol-position: right center;
  left: 0px;
  top: 0px;
  height: 18px;
  width: 18px;
}

</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="rows_slider">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Columns</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="cols_spin">
         <property name="styleSheet">
          <string notr="true">QSpinBox {
	max-height: 11px;
	min-height: 11px;
    min-width: 60px;
    max-width: 60px;
    padding-right: 10px;
    background-color: lightgray;
    padding-top: -2px;
    padding-bottom: -2px;
}

QSpinBox::up-button {
  subcontrol-origin: padding;
  subcontrol-position: right center;
  left: 7px;
  height: 18px;
  width: 18px;
}

QSpinBox::down-button {
  subcontrol-origin: content;
  subcontrol-position: right center;
  left: 0px;
  top: 0px;
  height: 18px;
  width: 18px;
}

</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="cols_slider">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Insert As Free Points</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
